Patch ace-builds to parse !secret yaml tag
---
diff --git a/source/assets/ace-builds/mode-yaml.js b/source/assets/ace-builds/mode-yaml.js
index 57ed248..262571b 100644
--- a/source/assets/ace-builds/mode-yaml.js
+++ b/source/assets/ace-builds/mode-yaml.js
@@ -31,6 +31,9 @@ var YamlHighlightRules = function () {
             }, {
                 token: "keyword.operator",
                 regex: "-\\s*(?=[{])"
+            }, {
+                token: "keyword.operator",
+                regex: "!\\w*\\s*",
             }, {
                 token: "string", // single line
                 regex: '["](?:(?:\\\\.)|(?:[^"\\\\]))*?["]'
diff --git a/source/assets/ace-builds/worker-yaml.js b/source/assets/ace-builds/worker-yaml.js
index 5875513..2207826 100644
--- a/source/assets/ace-builds/worker-yaml.js
+++ b/source/assets/ace-builds/worker-yaml.js
@@ -4963,6 +4963,11 @@ module.exports = new Type('tag:yaml.org,2002:timestamp', {
   represent: representYamlTimestamp
 });

+module.exports = new Type('!secret', {
+  kind: 'scalar',
+  construct: function (data) { return data !== null ? data : ''; }
+});
+
 },{"../type":16}],33:[function(require,module,exports){
 (function (global){
 var LARGE_ARRAY_SIZE = 200;
